src/xma/include/app/xmahw.h

```diff
@@ -33,6 +33,17 @@ typedef struct XmaHwSession
     uint32_t         reserved[4];
 } XmaHwSession;
 
+enum xma_cmd_state {
+  QUEUED = 1, //Submitted to XMA -> XRT
+  COMPLETED = 2, //Cmd has finished
+  ERROR = 3, //XMA or XRT error during submission of cmd
+  ABORT = 4, //XRT aborted the cmd; CU may or may not have received the cmd
+  TIMEOUT = 5, //XMA or XRT timeout waiting for cmd to finish
+  PSK_ERROR = 6, //PS Kernel cmd completed but with error return code
+  PSK_CRASHED = 7, //PS kernel has crashed
+  MAX // Always the last one
+};
```

Comment:
If this is a C++ header (not sure about that), then use enum class and lower case enumerators.  If this is a C header, then please prefix all enums so as not to conflict with braindead #defines in included header files.

Additionally if the enum is needed for end-user C-code, then  I strongly recommend having an enum class mirroring the classic enum.  E.g. within #ifdef __cplusplus have 
```
enum class xma_cmd_state : uint8_t {
queued = XMA_QUEUED;
completed = XMA_COMPLETED;
...
};
```
Write your implementation code, which is all C++, using this enum class.
Rename the classic C enum to `enum XmaCmdState` would of course be needed.

Reply:
Prefixed for end-user c-code and added enum class for internal use in c++ code